### ISSUE TITLE
Remove or update node-plop reference

### ISSUE THREAD

**Issue by awhitford** [plopjs/plop#453]
Is the link to [node-plop](https://github.com/plopjs/node-plop/) [here](https://github.com/plopjs/plop?tab=readme-ov-file#further-customization) still relevant?  Because the project has been archived, so it doesn't seem relevant anymore.

**Comment by crutchcorn** [comment]
Yes. `node-plop` was moved to this repo as part of a monorepo merge.
